### PR TITLE
Propose df TC translation.

### PR DESCRIPTION
**Pull request by SCgeeker**
There is the specific translation for degree of freedom.

### CODE REVIEW

File: terms/degrees-of-freedom.Rmd [hunk 1]
```diff
@@ -21,4 +21,10 @@ ggplot(dat, aes(x = df, y = p, color = as.factor(t))) +
 
 ```
```
Comment (SCgeeker) [inline terms/degrees-of-freedom.Rmd:23]: <div class="translation" lang="zh-Hant">

File: terms/degrees-of-freedom.Rmd [hunk 2]
```diff
@@ -21,4 +21,10 @@ ggplot(dat, aes(x = df, y = p, color = as.factor(t))) +
...
 
+<dfn>TRANSLATED TERM:自由度  有n個觀察值且已知樣本平均值，不能用已知估計值推測的可變異觀察值個數。</dfn>
 
+假設你想知道五位同班同學今年幾歲，而你事先知道五人平均20.1歲。樣本平均值是已知的估計值，你只能推測其中一位的真實年齡是20.1歲，但是其他四位回答的年齡可以是任何數值。這個例子的樣本有4個觀察值可自由變異，所以**自由度**是4。
+
+當你進行統計檢定如t檢定和F檢定，必須要知道正確的自由度(簡寫 *df*)。
+
+中文參考資料：林澤民(2017)。[統計學中算變異量為什麼要除以n-1？什麼是「自由度」？](http://blog.udn.com/nilnimest/92412101)
```
Comment (SCgeeker) [inline terms/degrees-of-freedom.Rmd:30]: Add </div> at the end.

**Review by SCgeeker — COMMENTED** [review]
Append html marks

**Comment by debruine** [comment]
Is there a translation for the figure caption? :

"How p-values vary depending on the degrees of freedom for specific t-values in a t-test."

**Pull request merged**